commit/core.pyx

```diff
@@ -75,7 +75,9 @@ cdef class Evaluation :
     cdef public A
     cdef public x
     cdef public CONFIG
-
+    cdef public confidence_map
```

Comment:
We have to be more clear about what is every variable. For instance, is `confidence_map` a NIFI file and `confidence_map_img` the corresponding data? And, also, do we need both variables as global variables?

Reply:
I used the same notation as `niiDWI` and `niiDWI_img`.
No, now we do not need both variables as global. Changed.